### PR TITLE
Tweak GHA configs

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

**Comment by fornewid** [comment]
Thanks for your pull request!
Please could you explain what you are changing for what reason?

**Comment by Goooler** [comment]
Catching Gradle with GBA, see https://github.com/gradle/gradle-build-action

**Comment by fornewid** [comment]
Thanks. Switching to the GBA sounds good to me. 👍 

But currently [Android Studio Electric Eel](https://developer.android.com/studio/releases) is stable.
So I want to keep it as minimum required Gradle version.
https://developer.android.com/studio/releases/gradle-plugin#updating-gradle

I wonder if we must increase the Gradle version?
If not, please down the Gradle version. :)

**Comment by Goooler** [comment]
Not necessary, but bumping the project's Gradle doesn't impact users' projects, we can keep the newer ones.

**Comment by fornewid** [comment]
I appreciate the suggestion, but I don't want to invest in incrementing the Gradle version now.
Please revert changes except `.github/workflows/` folder.

File: .github/workflows/publish-docs.yml [hunk 1]
```diff
@@ -7,31 +7,13 @@ on:
...
-      - name: Generate cache key
-        run: ./checksum.sh checksum.txt
-
-      - uses: actions/cache@v2
-        with:
-          path: |
-            ~/.gradle/caches/modules-*
-            ~/.gradle/caches/jars-*
-            ~/.gradle/caches/build-cache-*
-          key: gradle-${{ hashFiles('checksum.txt') }}
```
Comment (fornewid) [inline .github/workflows/publish-docs.yml:34]: Why was this code removed?

Comment (Goooler) [reply .github/workflows/publish-docs.yml:34]: No Gradle tasks are executed during docs publishing.

Comment (fornewid) [reply .github/workflows/publish-docs.yml:34]: `dokkaHtmlMultiModule` gradle task will be executed in [generate_docs.sh](https://github.com/fornewid/material-motion-compose/blob/main/generate_docs.sh).

Comment (Goooler) [reply .github/workflows/publish-docs.yml]: 803174b51ef91c196a411298d20980177fd454e9

**Review by fornewid — APPROVED** [review]
Thanks for your contribution. :bow:
Happy a nice weekend.

**Pull request merged**